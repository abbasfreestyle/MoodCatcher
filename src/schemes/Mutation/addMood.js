const addMood = `
  mutation(
    $username: String!,
    $date: AWSTimestamp!,
    $mood: Int!, 
    $comment: String
    ) {
    createMood(input:{
      username: $username
      date: $date,
      mood: $mood,
      comment: $comment
    }) {
      id
      username
      date
      mood
      comment
    }
  }
`;

export default addMood;
