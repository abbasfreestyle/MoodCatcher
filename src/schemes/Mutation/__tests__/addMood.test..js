import addMood from '../addMood';

describe('Schemes > Mutation > Add Mood', () => {
  test('should match the snapshot', () => {
    expect(addMood).toMatchSnapshot();
  });
});
