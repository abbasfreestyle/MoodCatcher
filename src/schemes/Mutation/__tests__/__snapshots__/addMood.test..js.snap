// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schemes > Mutation > Add Mood should match the snapshot 1`] = `
"
  mutation(
    $username: String!,
    $date: AWSTimestamp!,
    $mood: Int!, 
    $comment: String
    ) {
    createMood(input:{
      username: $username
      date: $date,
      mood: $mood,
      comment: $comment
    }) {
      id
      username
      date
      mood
      comment
    }
  }
"
`;
